src/main/kotlin/com/terraformation/backend/accelerator/api/CohortsController.kt

```diff
@@ -100,6 +106,34 @@ class CohortsController(
     return getCohort(cohortId)
   }
 
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{cohortId}/modules")
+  @Operation(summary = "List modules.")
```

Comment:
This is the same summary as the existing `listModules` endpoint; it'd be nice if the summaries said what they do differently.